.github/workflows/type.yml

```diff
@@ -9,7 +9,10 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/setup-node
+      uses: actions/checkout@v2
+      uses: ./.github/actions/setup-node
```

Comment:
this syntay isn't valid

Reply:
haha yeah I am trying to figure out whats wrong exactly